pkg/occlient/occlient.go

```diff
@@ -63,26 +64,17 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
-// CreateType is an enum to indicate the type of source of component -- local source/binary or git for the generation of app/component names
-type CreateType string
-
-const (
-	// GIT as source of component
-	GIT CreateType = "git"
-	// LOCAL Local source path as source of component
-	LOCAL CreateType = "local"
-	// BINARY Local Binary as source of component
-	BINARY CreateType = "binary"
-	// NONE indicates there's no information about the type of source of the component
-	NONE CreateType = ""
+var (
+	DEPLOYMENT_CONFIG_NOT_FOUND_ERROR_STR string = "deploymentconfigs.apps.openshift.io \"%s\" not found"
+	DEPLOYMENT_CONFIG_NOT_FOUND           error  = fmt.Errorf("Requested deployment config does not exist")
 )
 
 // CreateArgs is a container of attributes of component create action
 type CreateArgs struct {
 	Name            string
 	SourcePath      string
 	SourceRef       string
-	SourceType      CreateType
+	SourceType      util.CreateType
```

Comment:
can we rename `CreateType` to `SourceType`?

Reply:
Ack